src/lib/components/HoverCard/content.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang='ts' context='module'>
+    export type HoverCardContentProps = Exclude<ComponentProps<InstanceType<typeof Popper.Content>>, Required<PopperContentEventProps>>;
+</script>
+
+<script lang='ts'>
+    import {Popper} from '$lib/internal/components';
+	import type { PopperContentEventProps } from '$lib/internal/components/Popper/content.svelte';
+    import {getRootContext} from './root.svelte';
+
+    import type {ComponentProps} from 'svelte';
+
+    type $$Props = HoverCardContentProps;
+
+    const ctx = getRootContext();
+
+    export let side: NonNullable<$$Props['side']> = 'bottom';
+	export let sideOffset: NonNullable<$$Props['sideOffset']> = 0;
```

Comment:
We shouldn't modify the types coming from $$Props. Instead, it's preferable to modify HoverCardContentProps